html/pfappserver/root/static.alt/src/views/Configuration/_components/WrixLocationView.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <pf-config-view
+    :isLoading="isLoading"
+    :form="getForm"
+    :model="wrixLocation"
+    :vuelidate="$v.wrixLocation"
+    :isNew="isNew"
+    :isClone="isClone"
+    @validations="setValidations($event)"
+    @close="close"
+    @create="create"
+    @save="save"
+    @remove="remove"
+  >
+    <template slot="header" is="b-card-header">
+      <b-button-close @click="close" v-b-tooltip.hover.left.d300 :title="$t('Close [ESC]')"><icon name="times"></icon></b-button-close>
+      <h4 class="mb-0">
+        <span v-if="!isNew && !isClone">{{ $t('Wrix Location {id}', { id: id }) }}</span>
+        <span v-else-if="isClone">{{ $t('Clone Wrix Location {id}', { id: id }) }}</span>
+        <span v-else>{{ $t('New Wrix Location') }}</span>
+      </h4>
+    </template>
+    <template slot="footer"
+      scope="{isDeletable}"
+    >
+      <b-card-footer @mouseenter="$v.wrixLocation.$touch()">
+        <pf-button-save :disabled="invalidForm" :isLoading="isLoading">
+          <template v-if="isNew">{{ $t('Create') }}</template>
+          <template v-else-if="isClone">{{ $t('Clone') }}</template>
+          <template v-else-if="ctrlKey">{{ $t('Save &amp; Close') }}</template>
```

Comment:
Is the HTML entity encoding really necessary?

Reply:
I'll test. This is just from &quot ;habit&quot ;.